plugins/destination/clickhouse/typeconv/ch/types/decimal.go

```diff
@@ -5,41 +5,18 @@ import (
 	"strconv"
 
 	"github.com/apache/arrow/go/v15/arrow"
-	"golang.org/x/exp/constraints"
 )
 
 // https://clickhouse.com/docs/en/sql-reference/data-types/decimal
 func decimalType(decimalType arrow.DecimalType) (string, error) {
 	precision, scale := decimalType.GetPrecision(), decimalType.GetScale()
-	if scale > 76 {
```

Comment:
Should this be labeled a `fix`? Also is it a breaking change? And what does it fix?

Reply:
I think we don't have an issue here, as the arrow documents decimal types to have max precision as 76: https://arrow.apache.org/docs/r/reference/data-type.html
So I'd say this change is just removing an obsolete check